GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributions.References.cs

```diff
@@ -0,0 +1,30 @@
+﻿using GitFyle.Core.Api.Models.Foundations.Contributions;
+using Microsoft.EntityFrameworkCore;
+
+namespace GitFyle.Core.Api.Brokers.Storages
+{
+    internal partial class StorageBroker
+    {
+        void SetContributionReferences(ModelBuilder modelBuilder)
```

Comment:
Also rename the file to StorageBroker.Contributions.Configuration
```suggestion
        void AddContributionConfigurations(ModelBuilder modelBuilder)
```